projects/knora-ui/src/lib/viewer/pipes/knoradate.pipe.spec.ts

```diff
@@ -13,6 +13,28 @@ describe('KnoradatePipe', () => {
 
         const convertedDate = pipe.transform(date);
 
-        expect(convertedDate).toEqual('10/10/1993');
+        expect(convertedDate).toEqual('10.10.1993');
```

Comment:
your birthday? :-)

Reply:
You've found my easter egg :) I switched to using the date the Declaration of Independence was signed (7/4/1776, or 04.07.1776 for those who are not using FREEDOM formats) in other tests to ensure the order of day and month are correct.